lbrynet/conf.py

```diff
@@ -24,6 +24,9 @@
 
 ROOT_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
+# Compatibility Version of the wire protocol. Increasing this will break backward compatibility
+COMPAT_VERSION = 1
```

Comment:
i think this would be better in lbrynet.dht.constants

Reply:
yeah, makes sense. I'll move it